apps/explorer/src/explorer/styled.ts

```diff
@@ -38,6 +38,11 @@ export const GlobalStyle = createGlobalStyle`
     ${ScrollBarStyle}
   }
 
+  // TODO: remove this once we have a proper way to set global CSS variables for all apps
```

Comment:
cc/ @shoom3301

Reply:
There is a way - `GlobalCoWDAOStyles` in UI lib
@fairlighteth